edk2toolext/tests/test_web_dependency.py

```diff
@@ -211,24 +211,6 @@ def test_sha256_lowercase_single_file(self):
         if not os.path.isfile(file_path):
             self.fail("The downloaded file isn't there")
 
-    # Test that get_internal_path_root works the way we expect with a flat directory structure.
```

Comment:
Why drop the tests?

Reply:
The test was dropped because the function was removed.  The function was removed because no one called it in the code other than then test.